contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
+date: 2023-09-14
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/athlete-hog.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+  - Product engineers
+---
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
+
+In this post, Alexey shares his advice on how to get your growth team winning again.
+
+TLDR; there are 3 areas to investigate:
+
+1. Your team's velocity.
+2. Your experiment win rate.
+3. Your experiment win size.
+
+## Problem 1: Your team is moving too slowly
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of code quality being too high. "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
```

Comment:
The term here to use is Architecture Astronauts, I think: https://www.joelonsoftware.com/2001/04/21/dont-let-architecture-astronauts-scare-you/

Also
> slow velocity is usually a result of code quality being too high.
no, high code quality in the context of growth means the code is easy to iterate on.

The cause of slow velocity is over-engineering.